package.json

```diff
@@ -349,6 +350,11 @@
                 "title": "Set as Current Namespace",
                 "category": "Kubernetes"
             },
+            {
+                "command": "extension.vsKubernetesDashboard",
+                "title": "Open the Dashboard",
```

Comment:
super picky: just `Dashboard` or `Open Dashboard`?

Reply:
let's go with the second one here.